app/models/pet.rb

```diff
@@ -2,11 +2,21 @@ class Pet < ApplicationRecord
   belongs_to :shelter
   has_many :pet_applications, dependent: :destroy
   has_many :applications, through: :pet_applications
+  before_save :default_values
 
   def self.favpets(favorites)
     favorites.pets_favorite.keys.map do |petid|
       Pet.find(petid)
     end
   end
 
+  def default_values
+    self.adoption_status = "adoptable" if self.adoption_status.nil?
+    self.description = "Looking for a home" if self.description.nil?
+  end
```

Comment:
sets a default of both of these attributes since we don't have validations

Reply:
Interesting. I haven't seen before_save before. I will have to look into it and might have you give me a little explanation when we meet up again.